src/js/game/LevelMVC/LevelPlane.js

```diff
@@ -524,11 +526,22 @@ module.exports = class LevelPlane {
     let armBlock = new LevelBlock(pistonType);
     for (let i = blocksPositions.length - 1; i >= 0; --i) {
       let destination = [blocksPositions[i][0] + offsetX, blocksPositions[i][1] + offsetY];
+      let block = this.getBlockAt(blocksPositions[i]);
+      if (this.getBlockAt(destination) === undefined) {
+        console.log("break");
+      }
```

Comment:
nit: console.log

Reply:
I swear to god leaving these in my commits is my greatest weakness. I should just start eliminating the space between if and ( so that the linter won't even let me commit them.

Reply:
Pro tip: the linter we use is actually hierarchical, so if you put your own `.eslint` file in a directory above the project directory, it'll run that one as well as the official one. So you can have your own custom eslint that enforces a superset of rules, like:

```javascript
module.exports = {
  "rules": {
    "no-console": ["error"]
  }
}
```